catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,17 +18,22 @@
     <bean id="securityAttributes"
           class="org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes"/>
 
+    <reference id="systemHighAttributes"
+               interface="org.codice.ddf.security.SystemHighAttributes"
+               availability="mandatory"/>
+
     <bean id="defaultMarkingsPlugin"
           class="org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin">
         <cm:managed-properties
                 persistent-id="org.codice.alliance.catalog.plugin.defaultsecurity.DefaultSecurityAttributeValuesPlugin"
                 update-strategy="component-managed" update-method="update"/>
         <argument ref="securityAttributes"/>
+        <argument ref="systemHighAttributes"/>
         <argument>
             <map>
-                <entry key="classification" value="classification"/>
-                <entry key="releasability" value="CountryOfAffiliation"/>
-                <entry key="codewords" value="FineAccessControls"/>
+                <entry key="classification" value="Clearance"/>
```

Comment:
This should just be "classification" and not "Clearance". Clearance is the value that the "system user" has to assert against items with a classification.

Reply:
Updated the default value in the blueprint and metatype.